gcc/testsuite/rust/compile/torture/intrinsics-6.rs

```diff
@@ -22,5 +24,10 @@ fn main() -> i32 {
     let zero5 = unsafe { (1 << 5) - unchecked_shl(1, 5) };
     let zero6 = unsafe { (1 >> 5) - unchecked_shr(1, 5) };
 
+    let _pop1 = ctpop(42i32);
+    let _pop2 = ctpop(42u32);
+    let _pop3 = ctpop(42i128);
+    let _pop4 = ctpop(42u128);
```

Comment:
I think the test can be made to check the actual result. You could do something similar to the above, or you could even create a dedicated file that only checks ctpop.

Reply:
"gcc 14" is a bit too vague, as we're really tracking gcc master branch. If the builtin is missing, then we can't really merge it now I guess.